src/patterns/components/masthead/collection.yaml

```diff
@@ -15,6 +15,10 @@ restrictions:
     $sprk-masthead-narrow-height to that height.
   - You must duplicate your navigational items in the Masthead navigation HTML and in the Masthead narrow
     navigation HTML. There are two chunks of markup for the Masthead, one for large viewports and one for small.
+  - The narrow viewport expects three divs with the following classes to render properly
```

Comment:
I don't love how this is written. If anyone has suggestions on improving it, I'm open. I just think we should have something that says they have to have that third div.